make/jdk/src/classes/build/tools/cldrconverter/CLDRConverter.java

```diff
@@ -1195,6 +1201,25 @@ private static Map<String, String> generateRules(AbstractLDMLHandler<Map<String,
             }));
     }
 
+    private static Map<Locale, String> coverageLevelsMap() throws Exception {
+        // First, parse `coverageLevels.txt` file
+        var covMap = Files.readAllLines(Path.of(COVERAGELEVELS_FILE)).stream()
+            .filter(line -> !line.isBlank() && !line.startsWith("#"))
+            .map(line -> line.split(";", 3))
+            .collect(Collectors.toMap(
+                    a -> Locale.forLanguageTag(a[0].trim().replaceAll("_", "-")),
+                    a -> a[1].trim(),
+                    (v1, v2) -> v2,
```

Comment:
this will grab all listed entries.  Right now, that will get you basic and above.  Ideally you would include if `v1` is one of `(basic|moderate|modern|comprehensive)` — I'm proposing to add `core` or maybe even `undefined` locales in this list (though no consensus yet)

Reply:
OK, will filter explicitly for those levels.

Reply:
the whole set is `undetermined|core|basic|moderate|modern|comprehensive` you could use an enum and then you can select just one.